enclave-tls/README.md

```diff
@@ -75,6 +76,8 @@ For example:
 ./enclave-tls-server run  --crypto wolfcrypt
 ```
 
+Enclave TLS's log level can be adjusted through `-l` option. It has six levels: off, fatal, error, warn, info, and debug. The default value is error, i.e., showing the error log messages. The most verbose level is debug.
```

Comment:
Change to `can be set through `-l` option with 6 levels: .... The default level is `error`. The most verbose level is `debug``.